x-pack/plugins/security/public/plugin.ts

```diff
@@ -43,12 +45,15 @@ export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPlugi
     this.securityLicenseService = new SecurityLicenseService();
     const { license } = this.securityLicenseService.setup({ license$: licensing.license$ });
 
+    const authc = setupAuthentication({ http: core.http });
+
     this.navControlService.setup({
       securityLicense: license,
+      getCurrentUser: authc.getCurrentUser,
     });
 
     return {
-      anonymousPaths,
```

Comment:
__note:__ I can't think of any reason why we'd want to re-expose part of the core's own contract.

Reply:
I also took a look and I agree, seems unnecessary